static/js/composebox_typeahead.js

```diff
@@ -331,11 +333,10 @@ export function tokenize_compose_str(s) {
                 // maybe topic_list; let's let the stream_topic_regex decide later.
                 return ">topic_list";
         }
-    }
 
-    const timestamp_index = s.indexOf("<time");
-    if (timestamp_index >= 0) {
-        return s.slice(timestamp_index);
```

Comment:
I believe the correct fix is to move up this original block of code up (with some tweaks) into the switch checking each character right to left. 

We can have a case for `<` and then check using this code if `<time` is included in the string at position `i`, and return this last timestamp.